pkgs/development/python-modules/setuptools-dso/default.nix

```
@@ -6,12 +6,13 @@
 }:
 
 buildPythonPackage rec {
-  pname = "setuptools_dso";
+  pname = "setuptools-dso";
   version = "2.10";
   format = "pyproject";
```

```suggestion
  pyproject = true;
```
Could you please modernize this parameter?